docs/manual/config/keys.rst

```diff
@@ -118,8 +118,33 @@ Chords can also be chained to make even longer sequences.
         ])
     ]
 
-Modes can also be added to chains if required.
+Modes can also be added to chains if required. The following example
+demonstrates the behaviour when using the ``mode`` argument in chains:
 
+::
+
+    from libqtile.config import Key, KeyChord
+
+    keys = [
+        KeyChord([mod], "z", [
+            KeyChord([], "y", [
+                KeyChord([], "x", [
+                    Key([], "c", lazy.spawn("xterm"))
+                ], mode="inner")
+            ])
+        ], mode="outer")
+    ]
+
+After pressing Mod+z y x c, the "inner" mode will remain active. When pressing
+<escape>, the "inner" mode is left. Since the mode in between does not have
```

Comment:
Could you replace "left" with "exited" or something similar? I actually got confused because I read "left" as "remains" lol